tests/sentry/tasks/test_weekly_escalating_forecast.py

```diff
@@ -74,28 +87,30 @@ def test_multiple_groups_escalating_forecast(self, mock_query_groups_past_counts
         )
 
         run_escalating_forecast()
-        group_forecast = GroupForecast.objects.all()
-        assert len(group_forecast) == 3
-        for i in range(len(group_forecast)):
-            assert group_forecast[i].group in group_list
+        for i in range(len(group_list)):
+            fetched_forecast = EscalatingGroupForecast.fetch(
+                group_list[i].project.id, group_list[i].id
+            )
+            assert fetched_forecast.project_id == group_list[i].project.id
+            assert fetched_forecast.group_id == group_list[i].id
+            assert fetched_forecast.forecast == FORECAST_LIST_MOCK
 
     @patch("sentry.tasks.weekly_escalating_forecast.query_groups_past_counts")
-    def test_no_duped_groups_escalating_forecast(self, mock_query_groups_past_counts):
-        group_list = self.create_archived_until_escalating_groups(num_groups=3)
+    def test_update_group_escalating_forecast(self, mock_query_groups_past_counts):
+        group_list = self.create_archived_until_escalating_groups(num_groups=1)
 
         mock_query_groups_past_counts.return_value = self.get_mock_groups_past_counts_response(
             num_days=7, num_hours=2, groups=group_list
         )
 
         run_escalating_forecast()
-        group_forecast = GroupForecast.objects.all()
-        assert len(group_forecast) == 3
-        for i in range(len(group_forecast)):
-            assert group_forecast[i].group in group_list
+        first_fetched_forecast = EscalatingGroupForecast.fetch(
+            group_list[0].project.id, group_list[0].id
+        )
 
-        # Assert no duplicates when this is run twice
+        # Assert update when this is run twice
```

Comment:
This is probably fine, however, we should have it in mind if re-runs of the tasks happened often and we wanted the re-runs to be faster.